addons/base_import_module/models/ir_module.py

```diff
@@ -241,7 +273,8 @@ def _import_zipfile(self, module_file, force=False, with_demo=False):
                     mod_name = filename.split('/')[0]
                     is_data_file = filename in module_data_files[mod_name]
                     is_static = filename.startswith('%s/static' % mod_name)
-                    if is_data_file or is_static:
+                    is_translation = filename.startswith('%s/i18n' % mod_name)
```

Comment:
I think you only need to import `.po` files but not `.pot` files.
theoretically maybe also `%s/i18n_extra` but I don't think it will be used.

Reply:
Indeed, done

Reply:
Ha, since if `filename.startswith('%s/i18n_extra' % mod_name)` it must `filename.startswith('%s/i18n' % mod_name)`, the original code was correct but tricky